src/devices/ExplorerHat/Led.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.ExplorerHat
+{
+    /// <summary>
+    /// Represents a led light
+    /// </summary>
+    public class Led : IDisposable
+    {
+        private GpioController _controller;
+
+        /// <summary>
+        /// GPIO pin to which led is attached
+        /// </summary>
+        public int Pin { get; private set; }
+
+        /// <summary>
+        /// Led number on Hat
+        /// </summary>
+        public int Number { get; private set; }
+
+        /// <summary>
+        /// Led name on Hat
+        /// </summary>
+        public string Name { get; private set; }
+
+        /// <summary>
+        /// Gets if led is switched on or not
+        /// </summary>
+        /// <value></value>
+        public bool IsOn { get; set; }
```

Comment:
This is settable and public which is confusing. I'd do one of the things below (pick whichever feels more intuitive):
- remove On/Off methods and put their body in this property (possibly also rename property), so effectively something along the lines of:

```csharp
private bool _on = false;
public bool On
{
    get
    {
        return _on;
    }
    set
    {
        _on = value;
        _controller.Write(Pin, value);
    }
}
```

- remove property (or make it private)

Reply:
I've made set private... See previous comment to consider if it's useful...

Reply:
settable property is useful since you can do: `On = !On` but you can leave as is :)